feature-branch-repo-management-with-cli-example/README.md

```diff
@@ -0,0 +1,3 @@
+# Automate Repository Management with the JFrog CLI
```

Comment:
Let's change the title to *Repo Management GitHub Actions Example*.

Reply:
Done.